MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -15,18 +15,17 @@ internal static class Sdl
     public static int Minor;
     public static int Patch;
 
-    private static unsafe string GetString(IntPtr handle)
+    private static string GetString(IntPtr handle)
     {
         if (handle == IntPtr.Zero)
             return "";
 
-        var ptr = (byte*) handle;
-        while (*ptr != 0)
-            ptr++;
+        int len = 0;
+        while (Marshal.ReadByte(handle, len) != 0)
```

Comment:
It works just fine, I submitted the same code to Eto Forms several months ago and there have been no complaints so far... 

Why wouldn't it work?